### PR TITLE
Added build.sbt file and removed MicroService and MicroServiceBuild

### PR DESCRIPTION
**Pull request by Davidrymer97**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import Dependencies.appDependencies
+import TestPhases.oneForkedJvmPerTest2
+import Resolvers._
+import uk.gov.hmrc.DefaultBuildSettings.{ addTestReportOption, defaultSettings, scalaSettings }
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
```
Comment (arkowalski) [inline build.sbt]: do we need scoverageSettigs for our coverage to take effect?
e.g.
lazy val scoverageSettings = {
  import scoverage.ScoverageKeys
  Seq(
    // Semicolon-separated list of regexs matching classes to exclude
    ScoverageKeys.coverageExcludedPackages := """uk\.gov\.hmrc\.BuildInfo;.*\.Routes;.*\.RoutesPrefix;.*Filters?;MicroserviceAuditConnector;Module;GraphiteStartUp;.*\.Reverse[^.]*""",
    ScoverageKeys.coverageMinimum := 80.00,
    ScoverageKeys.coverageFailOnMinimum := true,
    ScoverageKeys.coverageHighlighting := true,
    parallelExecution in Test := false
  )
}

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+lazy val playSettings: Seq[Setting[_]] = Seq.empty
+
+name := "gform"
```
Comment (arkowalski) [inline build.sbt]: also add "    organization := "uk.gov.hmrc"

File: project/Resolvers.scala [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import sbt._
+
+object Resolvers {
```
Comment (arkowalski) [inline project/Resolvers.scala]: remove this, just put the content in build.sbt

File: project/TestPhases.scala [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import sbt.{ForkOptions, TestDefinition}
+import sbt.Tests.{Group, SubProcess}
+
+object TestPhases {
+
+  def oneForkedJvmPerTest2(tests: Seq[TestDefinition]) =
```
Comment (arkowalski) [inline project/TestPhases.scala]: why is there a "2" in the name ?

Comment (arkowalski) [reply project/TestPhases.scala]: also currently there is only one test phase, move this function into build.sbt and make it private

Comment (lutnos) [reply project/TestPhases.scala]: At one time, at least, there was an existing oneForkedJvmPerTest which we could not override.  If it works, then there is no reason why it should not be oneForkedJvmPerTest now

**Pull request closed without merging**